openshift/template.yaml

```diff
@@ -233,6 +233,8 @@ objects:
                 value: ${FREE_ADDRESSES_IMAGE}
               - name: DHCP_LEASE_ALLOCATOR_IMAGE
                 value: ${DHCP_LEASE_ALLOCATOR_IMAGE}
+              - name: AWS_SHARED_CREDENTIALS_FILE
```

Comment:
you don't need to define it in parameters?

Reply:
Only if it is a *template* parameter and should get different values on different `oc process <template.yaml>` of the template but I don't think it is the case here.